gulpfile.js

```diff
@@ -40,25 +40,113 @@ var output_directory = path.join('openslides', 'static');
+    dest: path.join(output_directory, 'css')
+  },
+  fonts_libs: {
+    src: mainBowerFiles({
+            filter: /\.(woff)|(woff2)$/
+         }),
+    dest: path.join(output_directory, 'fonts')
+  },
+  angular_chosen_img: {
+    src: path.join('bower_components', 'chosen', '*.png'),
+    dest: path.join(output_directory, 'css')
+  },
+  ckeditor_defaults: {
+    src: [path.join('bower_components', 'ckeditor', 'styles.js'),
+          path.join('bower_components', 'ckeditor', 'contents.css'),],
+    dest: path.join(output_directory, 'ckeditor')
+  },
+  ckeditor_skins: {
+    src: path.join('bower_components', 'ckeditor', 'skins', 'moono-lisa', '**', '*'),
+    dest: path.join(output_directory, 'ckeditor', 'skins')
+  },
+  ckeditor_plugins: {
+    src: [path.join('bower_components', 'ckeditor', 'plugins', 'clipboard', '**', '*'),
+          path.join('bower_components', 'ckeditor', 'plugins', 'colorbutton', '**', '*'),
+          path.join('bower_components', 'ckeditor', 'plugins', 'colordialog', '**', '*'),
+          path.join('bower_components', 'ckeditor', 'plugins', 'dialog', '**', '*'),
+          path.join('bower_components', 'ckeditor', 'plugins', 'find', '**', '*'),
+          path.join('bower_components', 'ckeditor', 'plugins', 'image', '**', '*'),
+          path.join('bower_components', 'ckeditor', 'plugins', 'justify', '**', '*'),
+          path.join('bower_components', 'ckeditor', 'plugins', 'link', '**', '*'),
+          path.join('bower_components', 'ckeditor', 'plugins', 'liststyle', '**', '*'),
+          path.join('bower_components', 'ckeditor', 'plugins', 'magicline', '**', '*'),
+          path.join('bower_components', 'ckeditor', 'plugins', 'pastefromword', '**', '*'),
+          path.join('bower_components', 'ckeditor', 'plugins', 'panelbutton', '**', '*'),
+          path.join('bower_components', 'ckeditor', 'plugins', 'showblocks', '**', '*'),
+          path.join('bower_components', 'ckeditor', 'plugins', 'specialchar', '**', '*'),
+          path.join('bower_components', 'ckeditor', 'plugins', 'sourcedialog', '**', '*'),
+          path.join('bower_components', 'ckeditor', 'plugins', 'table', '**', '*'),],
+    dest: path.join(output_directory, 'ckeditor', 'plugins')
+  },
+  ckeditor_lang: {
+    src: [path.join('bower_components', 'ckeditor', 'lang', 'en.js'),
+          path.join('bower_components', 'ckeditor', 'lang', 'de.js'),
+          path.join('bower_components', 'ckeditor', 'lang', 'pt.js'),
+          path.join('bower_components', 'ckeditor', 'lang', 'es.js'),
+          path.join('bower_components', 'ckeditor', 'lang', 'fr.js'),
+          path.join('bower_components', 'ckeditor', 'lang', 'cs.js'),
+          path.join('bower_components', 'ckeditor', 'lang', 'ru.js'),],
+    dest: path.join(output_directory, 'ckeditor', 'lang')
+  },
+  translations: {
+    src: path.join('openslides', 'locale', 'angular-gettext', '*.po'),
+    dest: path.join(output_directory, 'i18n')
+  }
+};
+
+function js() {
```

Comment:
I also prefer function expressions before function declarations. (See some opinion [here](https://stackoverflow.com/questions/1013385/what-is-the-difference-between-a-function-expression-vs-declaration-in-javascrip#comment41375064_3344397).)